src/utils.ts

```diff
@@ -77,6 +77,16 @@ export const getServicesFromLocalStorage = (): ServiceInfoFromLocalStorage[] =>
     return []
 }
 
+/**
+ * A method to check if user has interacted with the consent prompt and given consent information.
+ *
+ * @example hasUserInteracted()
+ */
+export const hasUserInteracted = (): boolean => {
+    const userInteraction = IS_BROWSER && localStorage?.getItem('uc_user_interaction')
+    return userInteraction === 'true'
```

Comment:
From looking at how `uc_user_interaction` is inserted, it might be enough to simply check that it exists, but this works too. 👍